content/TrueCommand/Administration/Settings/SAML.md

```diff
@@ -193,116 +190,111 @@ This procedure assumes the Windows administrator user is QE. Substitute your sys
     
 3. Configure the SAML app screen settings. 
 
    ![GAAddCustomSAMLAppDetailsCropped](/images/SAML/GAAddCustomSAMLAppDetailsCropped.jpg "GA Add Custom SAML App Details Cropped")
     
-   a. Type any name you want to use in the App Name field. This example uses tcsaml.
-   b. Upload any picture or avatar you want to use into the App icon area to identify the app in your google admin account.
-   c. Click **CONTINUE.** The Google Identity Provider details screen displays. 
+   a. Type any name you want to use in the **App Name** field. This example uses *tcsaml*.
+   b. Upload any picture or avatar you want to use into the **App icon** area to identify the app in your Google Admin account.
+   c. Click **CONTINUE.** to view the **Google Identity Provider** screen. 
 
-4. Click **CONTINUE**. The Service provider details screen displays.
+4. Click **CONTINUE** to view the **Service Provider** details screen.
 
 5. Configure the service provider details. 
     
    ![GAAddCustomSAMLAppAddServiceProviderDetailsCropped](/images/SAML/GAAddCustomSAMLAppAddServiceProviderDetailsCropped.jpg "GA Add Custom SAML App Add Service Provider Details Cropped")
     
-   a. Type or copy/paste the TrueCommand login URL http://IP:PORT/saml/acs in the into the ACS Url field. Where IP:PORT is your TrueCommand system IP and port address. 
-   b. Type any name you want to use into the Entity ID field. For example, truecommand-saml.
-   c. Type the https://IP:PORT/saml/helloURL into the Start URL field. Where IP:PORT is your TrueCommand system IP and port address.
-   d. Set Name ID format to PERSISTENT.
+   a. Type or copy/paste the TrueCommand login URL *http://IP:PORT/saml/acs* into the **ACS Url** field. *IP:PORT* is your TrueCommand system IP and port address. 
+   b. Type any name you want into the **Entity ID** field (ex. truecommand-saml).
+   c. Type the *https://IP:PORT/saml/helloURL* into the **Start URL** field. *IP:PORT* is your TrueCommand system IP and port address. 
+   d. Set **Name ID** format to **PERSISTENT**.
 
    ![GAAddCustomSAMLAppDetailsAddedCropped](/images/SAML/GAAddCustomSAMLAppDetailsAddedCropped.jpg "GA Add Custom SAML App Details Added Cropped")
     
-   e. Set Name ID to Basic Information > Primary Email.
-   f. Click **CONTINUE**. The Attribute Mapping screen displays.
+   e. Set Name ID to **Basic Information > Primary Email**.
+   f. Click **CONTINUE** to view the **Attribute Mapping** screen.
 
    ![GAAddCustomSAMLAppAttributeMappingCropped](/images/SAML/GAAddCustomSAMLAppAttributeMappingCropped.jpg "GAA dd Custom SAML App Attribute Mapping Cropped")
     
-   g. Enter the Attributes. Select the attribute using the Google Directory attributes dropdown menus and then type the attributes exactly as below into the App attribute fields:
+   g. Enter the Attributes. Select the attribute using the **Google Directory attributes** drop-down menus and then type the attributes exactly as below into the **App attributes** fields:
```

Comment:
Attributes is a screen in Google Admin, sugguest:

   g. Enter the **Attributes**. Select the attribute using the **Google Directory attributes** drop-down menus and then type the attributes exactly as below into the **App attributes** fields:

Reply:
Fixed